docs/src/release_notes.rst

```diff
@@ -3,6 +3,13 @@
 
 Release notes
 =============
+Unreleased
+----------
+Changed
+*******
+- :ref:`client` - Update docstring for :func:`Spotify.recommendations`
```

Comment:
Because we follow SemVer as described [here](https://pypi.org/project/tekore/), this should be under "fixed". Let's also include info that the documentation was changed *because* the API changed too. Maybe something like in the doc fixes in 3.4.0 or similar.

Reply:
Hold up, if this is not a change after all maybe it should be worded differently. But regardless, would be nice to have something that links the documentation change to the behavior, not that we simply updated documentation.

Reply:
Also "documentation" is better than "docstring" in my opinion 😅

Reply:
Maybe I shouldn't be nitpicking so much. This feels like I'm holding your hand, which is not the point. Sorry about that!